imports/ui/components/editor/components/localEditorComponent.js

```diff
@@ -43,17 +39,20 @@ class LocalEditorComponent extends CodemoEditor {
         <Row style={{ height: '7%' }} alignItems="center" justifyContent="space-between" padding="0 1rem">
           <p>{ this.getActiveFilename() }</p>
           <SaveFileButton getEditorContent={() => this.editor.getValue()} />
-          <TakeOverStreamButton getEditorContent={() => this.editor.getValue()} />
+          <TakeOverStreamButton localEditor={this.editor} />
```

Comment:
Not sure about this tweak. We're forcing `TakeOverStreamButton` to know about the monaco editor API if we pass the entire editor around.